code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm

```diff
@@ -0,0 +1,328 @@
+	health = 200
+	maxHealth = 200
+	armour_penetration = 15
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	move_to_delay = 3
+
+/mob/living/simple_animal/hostile/retaliate/goat/guard/pope
+	name = "Goat Pope"
+	desc = "For what is a God without a pope to spread their holy words"
+	icon_state = "goat_pope"
+	icon_living = "goat_pope"
+	icon_dead = "goat_pope_dead"
+	health = 100
+	maxHealth = 100
+	armour_penetration = 25
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	move_to_delay = 3
+	loot = list(/obj/item/clothing/head/goatpope)
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/Retaliate()
+	..()
+	if(stat == CONSCIOUS && prob(5))
+		visible_message("<span class='warning'>\The [src] bellows indignantly, with a judgemental gleam in his eye.</span>")
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/phase2/Retaliate()
+	set waitfor = FALSE
+	..()
+	if(spellscast < 5)
+		if(prob(5) && move_to_delay >= 3) //speed buff
+			spellscast++
+			visible_message("<span class='cult'>\The [src] shimmers and seems to phase in and out of reality itself!</span>")
+			move_to_delay = 1
+
+		else if(prob(5) && melee_damage_lower != 50) //damage buff
+			spellscast++
+			visible_message("<span class='cult'>\The [src]' horns grow larger and more menacing!</span>")
+			melee_damage_lower = 50
+
+		else if(prob(5)) //spawn adds
+			spellscast++
+			visible_message("<span class='cult'>\The [src] summons the imperial guard to his aid, and they appear in a flash!</span>")
+			var/mob/living/simple_animal/hostile/retaliate/goat/guard/master/M = new(get_step(src,pick(GLOB.cardinals)))
+			M.enemies |= enemies
+			var/mob/living/simple_animal/hostile/retaliate/goat/guard/G = new(get_step(src,pick(GLOB.cardinals)))
+			G.enemies |= enemies
+			G = new(get_step(src,pick(GLOB.cardinals)))
+			G.enemies |= enemies
+
+		else if(prob(5)) //EMP blast
+			spellscast++
+			visible_message("<span class='cult'>\The [src] disrupts nearby electrical equipment!</span>")
+			empulse(get_turf(src), 5, 2, 0)
+
+		else if(prob(5) && melee_damage_type == BRUTE && !special_attacks) //elemental attacks
+			spellscast++
+			visible_message("<span class='cult'>\The [src]' horns flicker with holy white flames!</span>")
+			melee_damage_type = BURN
+
+		else if(prob(5)) //earthquake spell
+			visible_message("<B><span class='danger'>\The [src]' eyes begin to glow ominously as dust and debris in the area is kicked up in a light breeze!!</span></B>")
+			stop_automated_movement = TRUE
+			if(do_after(src, 6 SECONDS, src))
+				var/health_holder = getBruteLoss()
+				visible_message("<B><span class='cult'>\The [src] raises its fore-hooves and stomps them into the ground with incredible force!!</span></B>")
+				explosion(get_step(src,pick(GLOB.cardinals)), -1, 2, 2, 3, 6)
+				explosion(get_step(src,pick(GLOB.cardinals)), -1, 1, 4, 4, 6)
+				explosion(get_step(src,pick(GLOB.cardinals)), -1, 3, 4, 3, 6)
+				stop_automated_movement = FALSE
+				spellscast += 2
```

Comment:
if the goat king casts earthquake twice, you've basically won the fight, as it won't be able to buff itself up with its first two spells.